restAssuredReqres/src/test/java/api/ReqresTests.java

```diff
@@ -1,166 +1,166 @@
-    public void checkSuccessfulRegisterTest() {
-        Specifications.installSpecifications(Specifications.requestSpec(URL),
-            Specifications.responseSpec(StatusCode.OK.getCode()));
-        register.put("email", "eve.holt@reqres.in");
-        register.put("password", "pistol");
-        Services.postRequest("api/register", register)
-            .then()
-            .body("id", equalTo(4))
-            .body("token", equalTo("QpwL5tke4Pnpja7X4"));
-    }
-
-    @Test(description = "REGISTER - UNSUCCESSFUL")
-    public void checkUnsuccessfulRegisterTest() {
-        Specifications.installSpecifications(Specifications.requestSpec(URL),
-            Specifications.responseSpec(StatusCode.BAD_REQUEST.getCode()));
-        register.put("email", "sydney@fife");
-        register.put("password", "");
-        Services.postRequest("api/register", register)
-            .then()
-            .body("error", equalTo("Missing password"));
-    }
-
-    @Test(description = "LOGIN - SUCCESSFUL")
-    public void checkSuccessfulLoginTest() {
-        Specifications.installSpecifications(Specifications.requestSpec(URL),
-            Specifications.responseSpec(StatusCode.OK.getCode()));
-        register.put("email", "eve.holt@reqres.in");
-        register.put("password", "cityslicka");
-        Services.postRequest("api/login", register)
-            .then()
-            .body("token", equalTo("QpwL5tke4Pnpja7X4"));
-    }
-
-    @Test(description = "LOGIN - UNSUCCESSFUL")
-    public void checkUnsuccessfulLoginTest() {
-        Specifications.installSpecifications(Specifications.requestSpec(URL),
-            Specifications.responseSpec(StatusCode.BAD_REQUEST.getCode()));
-        register.put("email", "peter@klaven");
-        register.put("password", "");
-        Services.postRequest("api/login", register)
-            .then()
-            .body("error", equalTo("Missing password"));
-    }
-
-    @Test(description = "DELAYED RESPONSE")
-    public void getDelayedListUsersTest() {
-        Specifications.installSpecifications(Specifications.requestSpec(URL),
-            Specifications.responseSpec(StatusCode.OK.getCode()));
-        Services.getRequest("/api/users?delay=3")
-            .then()
-            .body("data.id[0]", equalTo(1));
-    }
+package api;
+
+import enums.StatusCode;
+import java.util.HashMap;
+import java.util.Map;
+import org.testng.annotations.Test;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.notNullValue;
+
+
+public class ReqresTests {
+
+    private final static String URL = "https://reqres.in/";
+    private final Map<String, String> user = new HashMap<>();
+    private final Map<String, String> register = new HashMap<>();
+
+    @Test(description = "LIST USERS")
+    public void getListUsersTest() {
+        Specifications.installSpecifications(Specifications.requestSpec(URL),
+            Specifications.responseSpec(StatusCode.OK.getCode()));

```

Comment:
In my mind it will be better to extract method parameters and after that put them into method. It become more readable 
I mean:
variable1 = Specifications.requestSpec(URL);
variable2 = Specifications.responseSpec(StatusCode.OK.getCode());
Specifications.installSpecifications(variable1, variable2);

Please don't pay attention on variable names in example, it's just example :)